draft-ietf-rats-corim.md

```diff
@@ -2489,6 +2500,17 @@ The profile must specify how to compare the CBOR tagged Reference Value against
 
 Note that a Verifier may compare Reference Values in any order, so the comparison should not be stateful.
 
+## Concise Software Identifier Extension {#sec-coswid}
+
+A CoRIM may contain a CoSWID tag as specified in {{-coswid}}.
+The CoRIM profile allows for the specifier to provide meaning to extension points in the "Specification Required" range of the extension points `$version-scheme` and `$$coswid-extension`.
```

Comment:
Reword this sentence. I find it confusing. Perhaps it recites CoSWID rules for using extension points. If so, it can be assumed that this spec will follow IANA / CoSWID rules for extension points.